armi/tests/tutorials/anl-afci-177-blueprints.yaml

```diff
@@ -111,9 +111,9 @@ blocks:
             Tinput: 25.0
             Thot: 450.0
             axialPitch: 30.0
-            helixDiameter: 0.84825
+            helixDiameter: 0.777
```

Comment:
This is updated to 0.777 cm because the reflector slug OD is 0.6962 cm and not the same as the clad OD dimensions for the fuel. This leads to a smaller pitch.

Reply:
I wonder where that reflector pin OD came from. Maybe later we should just make that equal to the fuel pin OD just to make it less strange.